test/unit/provisioning/device-configGroup-utils_test.js

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
Is test/unit/provisioning/device-configGroup-utils_test.js a "parallel" of some existing service-based test?

Reply:
Yes, It is parallel of test/unit/provisioning/device-group-utils_test.js